modules/ROOT/attachments/cloud-dataplane-api.yaml

```diff
@@ -1227,7 +1227,7 @@ components:
         tags:
           additionalProperties:
             type: string
-          description: Optional list of tags to attach to a pipeline.
+          description: Optional lList of tags to attach to a pipeline.
```

Comment:
@kbatuigas This automated one keeps coming through, and keeps suggesting the same two edits, which I think are both mistakes. Any idea why?